enhancements/cluster-scope-secret-volumes/csi-driver-host-injections.md

```diff
@@ -512,4 +380,77 @@ And some details from our PV based prototyping with the hostpath examples (again
 
 ## Infrastructure Needed
 
-N/A
+- GitHub repositories for the CSI driver and operator
+- Add the CSI driver and operator to the OpenShift release payload (ART)
+
+## Appendix
```

Comment:
I copied the deeper R&D and proof of concept results here. I think we should keep this information for future reference, however it gets in the way of the proposal if left in higher sections.